network/arp_packet.ksy

```diff
@@ -0,0 +1,28 @@
+meta:
+  id: arp_packet
+  title: ARP network packet
+  endian: be
+seq:
+  - id: hardware_type
+    type: u2be
```

Comment:
This one can be just `u2`, as whole packet exclusively uses big-endian integers.